fastlane/actions/update_version_name.rb

```diff
@@ -0,0 +1,60 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      UPDATE_VERSION_NAME_CUSTOM_VALUE = :UPDATE_VERSION_NAME_CUSTOM_VALUE
+    end
+
+    class UpdateVersionNameAction < Action
+      def self.run(params)
+        # fastlane will take care of reading in the parameter and fetching the environment variable:
+        UI.message "Parameter API Token: #{params[:name]}"
+        newValue = params[:name]
+        gradleFile = File.read('app/build.gradle')
+        update = gradleFile.gsub(/(versionName\W)"(.+)"/) do |match|
+          "#{$1}\"#{newValue}\""
+        end
+        File.open('app/build.gradle', 'w') {|file| file.puts update }
+      end
```

Comment:
Does this actually answer my question above about manually updating the version number?

Reply:
Yes, if I understood your question. This action updates the version number for you, eliminating the need to manually maintain the version.